### PR TITLE
update tc_2018 due to bz2000415 fixed

### PR DESCRIPTION
**Pull request by Junefen**
Unexpected closed this pr, so resubmit once.
After the bugs fix, virt-who will fail to start with server=xxx for ahv mode.

```
# pytest tests/tier2/tc_2018_validate_server_option_in_etc_virtwho_d.py 
======================== test session starts ========================
platform linux -- Python 3.9.9, pytest-6.2.5, py-1.10.0, pluggy-1.0.0
rootdir: /root/workspace/virtwho-ci
collected 1 item                                                                                                                                 

tests/tier2/tc_2018_validate_server_option_in_etc_virtwho_d.py .                   [100%]
```

### CODE REVIEW

File: tests/tier2/tc_2018_validate_server_option_in_etc_virtwho_d.py [hunk 1]
```diff
@@ -35,17 +35,23 @@ def test_run(self):
...
+            res1 = self.op_normal_value(data, exp_error="1|2|3", exp_thread=1, exp_send=0)
         res2 = self.msg_validation(rhsm_output, msg_list, exp_exist=True)
         results.setdefault('step2', []).append(res1)
         results.setdefault('step2', []).append(res2)
 
         logger.info(">>>step3: server option is 红帽€467aa value")
         self.vw_option_update_value(option_tested, '红帽€467aa', config_file)
         data, tty_output, rhsm_output = self.vw_start()
-        if "RHEL-7" not in compose_id and "esx" in hypervisor_type:
+        if "esx" in hypervisor_type or "ahv" in hypervisor_type:
```
Comment (hkx303) [inline tests/tier2/tc_2018_validate_server_option_in_etc_virtwho_d.py]: You could try:
```suggestion
        if hypervisor_type in ['esx', 'ahv']:
```

**Review by hkx303 — APPROVED** [review]
Thanks for the update, a small advice :)

File: tests/tier2/tc_2018_validate_server_option_in_etc_virtwho_d.py (hunk 1 above)
Comment (Junefen) [reply tests/tier2/tc_2018_validate_server_option_in_etc_virtwho_d.py]: Updated.

**Comment by Junefen** [comment]
It's such an old pr because my delay update, will merge it by myself because I have tested it again after the updates.

**Pull request merged**